fandogh_cli/image_commands.py

```diff
@@ -57,10 +57,13 @@ def list_images():
 
 
 def show_image_logs(image_name, version):
+    image_offset = 0
+
     if not image_name:
         image_name = get_project_config().get('image.name')
     while True:
-        response = get_image_build(image_name, version)
+        response = get_image_build(image_name, version, image_offset)
+        image_offset = len(response.get('logs').split('\n'))
```

Comment:
you don't need to find the offset in this way. 
It should be part of the response you are getting